tests/unit/modules/test_btrfs.py

```diff
@@ -410,12 +410,13 @@ def test_subvolume_create(self, subvolume_exists):
         salt_mock = {
             'cmd.run_all': MagicMock(return_value={'recode': 0}),
         }
+        expected_path = os.path.join('/mnt', 'var')
```

Comment:
I think the module is intended for Linux. Since we're just mocking I left it as is. We could just skip the tests on Windows. Though there is a btrfs driver for Windows... I don't know how often it is used or if we should support it.

Reply:
https://github.com/maharmstone/btrfs